server/schedulers/utils.go

```diff
@@ -113,3 +114,138 @@ const (
 func newTaintCache() *cache.TTLUint64 {
 	return cache.NewIDTTL(taintCacheGCInterval, taintCacheTTL)
 }
+
+// ScorePair stores storeID and record of a store.
+type ScorePair struct {
+	storeID uint64
+	score   float64
+}
+
+// NewScorePair returns a ScorePair.
+func NewScorePair(storeID uint64, score float64) *ScorePair {
+	return &ScorePair{
+		storeID: storeID,
+		score:   score,
+	}
+}
+
+// GetStoreID returns the storeID.
+func (s *ScorePair) GetStoreID() uint64 {
+	return s.storeID
+}
+
+// GetScore returns the score.
+func (s *ScorePair) GetScore() float64 {
+	return s.score
+}
+
+// SetScore sets the score.
+func (s *ScorePair) SetScore(score float64) {
+	s.score = score
+}
+
+// ScorePairs is a named type for []*ScorePair with helper methods.
+type ScorePairs []*ScorePair
```

Comment:
Please use `ScorePairSlice` or `SortScorePairs`. `ScorePairs` and `ScorePair` are too similar.